src/Plugin/MergePlugin.php

```diff
@@ -81,7 +89,11 @@ public function preAutoloadDump(ScriptEvent $event): void
     {
         $rootPackage = $this->composer->getPackage();
         $this->mergeAutoloads($rootPackage);
-        $this->mergeAutoloadOverrides($rootPackage);
+
+        $overrides = $rootPackage->getExtra()['overrides'] ?? true;
+        if ($overrides) {
```

Comment:
```suggestion
        if ($rootPackage->getExtra()['overrides'] ?? true) {
```
die lokale Variable hier und in Zeile 44 wird nur jeweils genau einmal benutzt, ich würde sie inlinen